imnodes.h

```diff
@@ -58,6 +65,7 @@ struct Style
 
     float pin_radius = 4.0f;
     float pin_hover_radius = 10.0f;
+    float pin_offest = 10.0f;
```

Comment:
typo, pin_offest -> pin_offset.

Also, I'd like to keep the default behavior of the library mostly the same as before. Would it be possible to default this to zero?

Reply:
Fine, a new PR will come and a new interface to set the value of pin_offset.

Reply:
Don't need a new interface as have imnodes::GetStyle().

Reply:
Yep, exactly 🙂 I didn't add `pin_radius` and `pin_hover_radius` to the StyleFlags/Push/PopStyle, since I figured it didn't make a lot of sense to have a per-pin configuration for the radius/hover radius. The same is true for the pin offset!